[Draft] Clarify serialization and discoverability (#40)
frictionlessdata/datapackage: content/docs/specifications/data-resource.md

}
```

## Descriptor

On logical level, Data Resource descriptor is represented by a data structure. The data structure `MUST` be a JSON-serializable `object` as defined in [RFC 4627](http://www.ietf.org/rfc/rfc4627.txt).

On physical level, Data Resource descriptor is represented by a file. A data producer `MAY` use any suitable serialization format. A data consumer `MUST` support JSON serialization format and `MAY` support other serialization formats like YAML or TOML.

The above states that JSON is the only serialization format that `MUST` be used for publishing a Data Resource while other serialization formats can be used in projects or systems internally if supported by corresponding implementations.

> This specification does not define any discoverability mechanisms making a serialized Data Resource be referenced only directly by its URI.

Reviewer: ```suggestion
This specification does not define any discoverability mechanisms. Any URI can be used to directly reference a serialized Data Resource. 
```
